resteasy-dependencies-bom/pom.xml

```diff
@@ -745,6 +746,21 @@
                 <artifactId>json-patch</artifactId>
                 <version>${version.json.patch}</version>
             </dependency>
+            <dependency>
```

Comment:
These deps could probably stay here, there are more deps which are not sorted alphabetically. @asoldano wdyt?